tests/api/controllers/range_finder_horizontal_spherical_projection/range_finder_horizontal_spherical_projection.c

```diff
@@ -27,7 +27,7 @@ int main(int argc, char **argv) {
     float sub_cameras_junction_depth = wb_range_finder_image_get_depth(image, width, x1, height - 1);
     float sub_camera_center_depth = wb_range_finder_image_get_depth(image, width, x2, height - 1);
     ts_assert_double_in_delta(
-      sub_cameras_junction_depth, sub_camera_center_depth, 0.01,
+      sub_cameras_junction_depth, sub_camera_center_depth, 0.1,
```

Comment:
Depth sensors (e.g. https://www.photoneo.com/products/phoxi-scan-xl/) have an accuracy of 0.5mm at 2-3 meters. Even cheap depth sensors (like RealSense D435) have an accuracy of up to 4cm. Therefore, we should be careful with these thresholds.

Reply:
> Depth sensors (e.g. https://www.photoneo.com/products/phoxi-scan-xl/) have an accuracy of 0.5mm at 2-3 meters. Even cheap depth sensors (like RealSense D435) have an accuracy of up to 4cm. Therefore, we should be careful with these thresholds.

This so big difference was in fact due to the fact of those square corners as you can see on the first picture (left). This is not a regression, it is also present in `develop` (see second picture), but in `develop` there is an additional issue: there is an angle offset between the theoretical location of the lidar layers according to Webots interface and the actual place where the bottom layer is shown (1st picture, right).

![error](https://user-images.githubusercontent.com/38250944/124087303-29e55180-da52-11eb-9880-ce8688d93f56.png)
Projection test: This PR (left), `develop` (right)
![range_finder_horizontal_spherical_projection](https://user-images.githubusercontent.com/38250944/124087336-2fdb3280-da52-11eb-8610-ca60cdad7d77.png)
Same configuration but with the `Lidar` a little closer to the ground (`develop`) to show the square corners.

Which means that to have a fair comparison, I changed again the test so that instead of looking at the 79th layer, we look at the 70th which is located above this square corners. This allows then to put the maximum difference between a sub-camera's center and junction depths to 4cm.

The issue with the square corners could be addressed in another pull request.